pkg/schemadsl/lexer/lex_test.go

```diff
@@ -126,6 +126,22 @@ var lexerTests = []lexerTest{
 		tEOF,
 	}},
 
+	{"relation with caveat", "/* foo */relation viewer: user with somecaveat\n", []Lexeme{
```

Comment:
should we add more lexer tests for permutations here?
- `user:* with somecaveat`
- `user#rel with somecaveat`
- `user#with with somecaveat`
- `user#rel with with`
- ...

Reply:
Added some of them, although it shouldn't matter the order of tokens in the lexer